addons/website_event_sale/models/sale_order.py

```diff
@@ -30,7 +30,9 @@ def _website_product_id_change(self, order_id, product_id, qty=0, **kwargs):
         else:
             product = self.env['product.product'].browse(product_id)
             if product.event_ticket_ids:
-                event_ticket_id = product.event_ticket_ids[0].id
+                # When a customer adds an event product directly to the cart we don't want to assign tickets from
+                # ended events as it's confusing and leads to errors.
```

Comment:
I think it's self-explanatory enough, wouldn't add a comment for that
Even if we do, we already know we're in the flow of adding things to the cart so I'd at least shorten it to not mention that :)

Reply:
Ok, removed :)